README.md

```diff
@@ -40,10 +49,10 @@ Stamps have a method called `.compose()`:
 stamp.compose(...args?: stamp|descriptor[]): stamp
```

Comment:
Sorry, that's wrong. It was already correct:

```js
stamp.compose(...args?: stamp|descriptor[]): stamp
```

This means `...args` is an Array that can contain a mix of stamps or descriptors, but it would be more clear if we wrote it like this:

```js
stamp.compose(...args?: composable[]): stamp
```